src/Settings/settings.json

```diff
@@ -241,6 +241,9 @@
         "type": "int",
         "default": 1000,
         "param": "QVariantList {100, 36000000}",
+        "depends": [
+            "Auto Save"
```

Comment:
an exception may be thrown in function `main` which should not throw exceptions [bugprone-exception-escape,-warnings-as-errors]
```cpp
int main()
    ^
```
Reported as error by clang-tidy.